README.md

```diff
@@ -77,7 +77,7 @@ Processing back ends can be switched via menu `Settings/Back end`.
 
   - The current selection can be changed at any time by dragging the mouse with left button pressed. Changing the selection cancels all previously started processing (if any).
 
-  - The smaller the selection, the faster the processing. For fine-tuning of L–R deconvolution’s *sigma*, it is recommended to use moderately small areas; this way moving the *sigma* slider will show the results almost instanteneously. The refreshing is slightly slower when the current zoom factor is not 100%.
+  - The smaller the selection, the faster the processing. For fine-tuning of L–R deconvolution's *sigma*, it is recommended to use moderately small areas; this way moving the *sigma* slider will show the results almost instantaneously. The refreshing is slightly slower when the current zoom factor is not 100%.
```

Comment:
Thanks!
But please bring back all the apostrophe characters (U+2019); I've added them on purpose ;)

Reply:
Lol! Sublime Text does not like them hence why I removed them :) Will put them back.